ethcore/src/client/client.rs

```diff
@@ -1641,7 +1641,7 @@ impl BlockChainClient for Client {
 		self.chain.read().logs(blocks, |entry| filter.matches(entry), filter.limit)
 	}
 
-	fn filter_traces(&self, filter: TraceFilter) -> Option<Vec<LocalizedTrace>> {
+	fn filter_traces(&self, filter: TraceFilter, after: Option<&TraceId>, count: Option<u64>) -> Option<Vec<LocalizedTrace>> {
```

Comment:
Why not making it part of `TraceFilter`?

Reply:
Using separate parameters breaks backward compatibility, we already support `limit` parameter for `eth_getLogs` filter object and that would be inline with that.

Would be good if those parameters could be omitted (using `Trailing`), but we don't suport two optional parameters.

IMHO would be better to make it part of `TraceFilter` object.